### DISCUSSION TITLE
Python ile eklenti geliştirmek için / Writing extensions with Python

### DISCUSSION THREAD

**Discussion by hwpplayer1** [limanmys/core#153] [Eklenti Geliştirme]
I want to learn about Python, if it is suitable the needs of this project

Python'ın bu projeyle uyumlu olup olmadığını merak ediyorum

Ansible var ise herhalde ben de bir Ansible yazmasam da Python kullanabilirim veya kendi Python'ımla olabilirim gibi 

[Ansible Wikipedia Tanıtımı](https://en.wikipedia.org/wiki/Ansible_(software))

**Comment by aliorhun** [comment]
Bu konuda aşağıdaki makaleyi inceleyebilirsiniz. Python ile eklenti geliştirme yapabilirsiniz.  

https://dev.to/liman/python-betigi-i-le-eklenti-gelistirme-5240

**Reply by hwpplayer1** [reply]
Teşekkürler
@aliorhun
